ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -25,25 +72,17 @@ public final class Atomic<Value> {
 	}
 	
 	/// Initializes the variable with the given initial value.
-	public init(_ value: Value) {
+	public init(_ value: Value, mutex: MutexType = PosixThreadMutex()) {
```

Comment:
I don't think I see why we'd want this to be a public API: why would anyway want to pick a particular mutex implementation?